app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class CardsMailer < ApplicationMailer
+  default from: 'flashcards.se@gmail.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.